tests/Makefile

```diff
@@ -180,7 +180,7 @@ legacy : CPPFLAGS+= -I$(ZSTDDIR)/legacy
 legacy : $(ZSTD_FILES) $(wildcard $(ZSTDDIR)/legacy/*.c) legacy.c
 	$(CC)      $(FLAGS) $^ -o $@$(EXT)
 
-decodecorpus	: $(filter-out $(ZSTDDIR)/compress/zstd_compress.c, $(wildcard $(ZSTD_FILES))) decodecorpus.c
+decodecorpus	: $(filter-out $(ZSTDDIR)/compress/zstd_compress.c, $(wildcard $(ZSTD_FILES))) $(ZDICT_FILES) decodecorpus.c
 	$(CC)      $(FLAGS) $^ -o $@$(EXT) -lm
```

Comment:
Actually, I was thinking the tests script could be located directly in `Makefile`.
You can have a look at `lz4` project, in `tests/Makefile`, for guidance.

The reason we have `playTests.sh` in `zstd` is that some (rare) platforms don't have `make` pre-installed, but their owner nonetheless want to run some tests.